tests/robustness/report/report.go

```diff
@@ -42,7 +43,8 @@ func testResultsDirectory(t *testing.T) string {
 	if err != nil {
 		panic(err)
 	}
-	path, err := filepath.Abs(filepath.Join(resultsDirectory, strings.ReplaceAll(t.Name(), "/", "_")))
+	path, err := filepath.Abs(filepath.Join(
+		resultsDirectory, strings.ReplaceAll(t.Name(), "/", "_"), fmt.Sprintf("%v", time.Now().UnixNano())))
```

Comment:
This will create 2 level directory hierarchy like `/tmp/TestRobustnessExploratory_Kubernetes_HighTraffic_ClusterOfSize3/38712372198371`

Not what I had in mind, still should be good enough.